### PR TITLE
Stripe Documentation Update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Stripe documentation updates 

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Stripe Cloud Extract documentation updates](https://www.notion.so/rudderstacks/Stripe-Cloud-Extract-documentation-update-95bb40c2cbee472b9c1dcc607db5e72b)

### CODE REVIEW

**Comment by pfd** [comment]
I'm curious about this customer experience. I tested setting up a Stripe integration from RudderStack, and the "Connect with Stripe" button led me down a path to "Connect with Blendo." 

New customers might not have any idea what Blendo is. Is this something we should clarify or change the referral URL or something? 
Screen Shot 2022-05-04 at 7 10 16 PM

**Review by pfd — APPROVED** [review]
👍🏻  Added a couple of edits and a question about Blendo (should not block release)

File: cloud-extract-sources/stripe.mdx [hunk 1]
```diff
@@ -1,88 +1,86 @@
...
 description: >-
-  Step-by-step guide to ingest your payment data from your Stripe account into
-  RudderStack.
+  Step-by-step guide on ingesting data from Stripe into RudderStack.
 ---
 
-# Stripe
+# Stripe Source
 
 [Stripe](https://stripe.com/) is an online payment processing system for businesses. It offers cutting-edge software and APIs that allows thousands of businesses to manage their business payments online. Retailers, subscription businesses, software platforms, and online marketplaces all use Stripe for online payment.
```
Comment (ishachopra0401) [inline cloud-extract-sources/stripe.mdx:9]:
```suggestion
[Stripe](https://stripe.com/) is an online payment processing system for businesses. It offers cutting-edge software and APIs that allows thousands of businesses to manage their business payments online. Many retailers, subscription businesses, software platforms, and online marketplaces use Stripe for online payment.
```

**Review by ishachopra0401 — APPROVED** [review]

**Comment by ameypv-rudder** [comment]
As per the discussion with @RanjeetMishra - we can proceed for now, and take up the Blendo -> RudderStack logo change as a part of the next iteration. cc @pfd

**Pull request merged**